package.json

```
@@ -22,6 +22,7 @@
     "sauce-testrunner-utils": "^0.3.0",
     "saucelabs": "^4.6.5",
     "testcafe": "1.11.0",
+    "testcafe-browser-provider-ios": "^0.3.0",
```

Imho, specifying it with a caret doesn't give us any benefits, since versions are always pinned down at release and we'd never want them to differ between releases unless explicitly updated. This is true for all the other dependencies btw. But I'd rather avoid adding new ones in this manner :)

```suggestion
    "testcafe-browser-provider-ios": "0.3.0",
```